lib/java/com/google/android/material/internal/ForegroundLinearLayout.java

```diff
@@ -48,7 +48,7 @@ public class ForegroundLinearLayout extends LinearLayoutCompat {
 
   protected boolean mForegroundInPadding = true;
 
-  boolean foregroundBoundsChanged = false;
+  boolean foregroundBoundsChanged = true;
```

Comment:
Yeah, I know. I just wanted to avoid future surprises in case the logic becomes more complicated and this flag is forgotten again 😄.
Anyway, thanks for the comment.